test/restDocs/test.spec.ts

```diff
@@ -1,20 +1,29 @@
-// cspell:word colordiff
-
-import assert from 'node:assert';
-import {execSync, spawnSync} from 'node:child_process';
-import {readFileSync} from 'node:fs';
+import {execSync} from 'node:child_process';
+import {readdirSync, readFileSync} from 'node:fs';
 import {dirname, join} from 'node:path';
 import {fileURLToPath} from 'node:url';
 import {App} from '../../src/app.js';
 import {App as DtApp} from '../../src/dt/app.js';
 import {RestDescription} from '../../src/discovery.js';
 import {getPackageName, getPackageNameLegacy} from '../../src/utils.js';
+import {toMatchSpecificSnapshot as _toMatchSpecificSnapshot} from 'jest-specific-snapshot';
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
+expect.extend({
+  toMatchSpecificSnapshot(received, snapshotFile) {
+    return _toMatchSpecificSnapshot.call(
+      this,
+      received.toString(),
+      snapshotFile,
+      this.currentTestName || ''
+    ) as never;
```

Comment:
` as never` shouldn't be necessary here.

We have two options:
1. Change `node_modules/@types/jest-specific-snapshot/index.d.ts` using `npx patch-package`
2. Just `import 'jest-specific-snapshot'`

If we go with 2nd option, we'll get:
- slightly worse snapshot names 
- we'll have to add a helper function `const readFileSyncAsUTF8 = (path: string) => readFileSync(path, 'utf-8');`
But on the upside, we will avoid two workarounds:
- no need to patch types pacakge
- no need to override `toMatchSpecificSnapshot` functionality

Both workarounds might cost us in the future as we will likely have to change them as our dependencies update.

So I think it'd be best to use the second approach, what do you think?

If you will go with that approach, you'll need to delete the snapshots folder and re-run tests to generate snapshots with updated names. Thank you!

![image](https://user-images.githubusercontent.com/7756211/192074827-4f48efdb-41bf-43df-9c86-44e4cf0e4e21.png)